tests/linalg_test.py

```diff
@@ -1440,5 +1440,43 @@ def test_tridiagonal_solve(self, dtype):
     A[[0, 1], [1, 2]] = du[:-1]
     np.testing.assert_allclose(A @ X, B, rtol=1e-6, atol=1e-6)
 
+  @parameterized.named_parameters(
+        jtu.cases_from_list({
+            "testcase_name":
+            "_shape={}".format(jtu.format_shape_dtype_string(shape, dtype)),
+            "shape": shape, "dtype": dtype
+        } for shape in [(4, 4), (15, 15), (50, 50), (100, 100)]
+                            for dtype in float_types + complex_types))
+  @jtu.skip_on_devices("gpu", "tpu")
+  def testSchur(self, shape, dtype):
+      if jax._src.lib.version < (0, 1, 72):
+          self.skipTest("Schur LAPACK wrapper only implemented for jaxlib versions >= (0, 1, 72)")
+      rng = jtu.rand_default(self.rng())
+      args_maker = lambda: [rng(shape, dtype)]
+
+      self._CheckAgainstNumpy(osp.linalg.schur, lax.linalg.schur, args_maker)
+      self._CompileAndCheck(lax.linalg.schur, args_maker)
+
+  @parameterized.named_parameters(
+      jtu.cases_from_list({
+          "testcase_name":
+          "_shape={}".format(jtu.format_shape_dtype_string(shape, dtype)),
+          "shape": shape, "dtype": dtype
+      } for shape in [(2, 2), (4, 4), (15, 15), (50, 50), (100, 100)]
+                          for dtype in float_types + complex_types))
+  @jtu.skip_on_devices("gpu", "tpu")
+  def testSchurBatching(self, shape, dtype):
+      if jax._src.lib.version < (0, 1, 72):
+          self.skipTest("Schur LAPACK wrapper only implemented for jaxlib versions >= (0, 1, 72)")
+      rng = jtu.rand_default(self.rng())
+      batch_size = 10
+      shape = (batch_size, ) + shape
+      args = rng(shape, dtype)
+      Ts, Ss = vmap(lax.linalg.schur)(args)
+      for i in range(batch_size):
+          self.assertAllClose(
+              (jnp.matmul(jnp.matmul(Ss[i], Ts[i]), jnp.conjugate(Ss[i].T))),
+              args[i], atol=1e-4)
```

Comment:
Perhaps it would be more clear to do something like:
```python
f = vmap(lambda S, T: S @ T @ jnp.conj(S.T))
self.assertAllClose(f(Ss, Ts), args)
```